Make R identify -lflexiblas as being for FlexiBLAS, so that it is not reported as a generic BLAS
Patch by Simon Branford (University of Birmingham)
--- configure.orig	2021-06-03 17:38:35.653766000 +0100
+++ configure	2021-06-03 17:40:25.970132472 +0100
@@ -55457,6 +55457,7 @@
     *-latlas*) r_blas=ATLAS ;;
     *-lgoto*) r_blas=Goto ;;
     *-lopenblas*) r_blas=OpenBLAS ;;
+    *-lflexiblas*) r_blas=FlexiBLAS ;;
     *-lacml*) r_blas=ACML ;;
     *-lmkl*) r_blas=MKL ;;
     *-lsgemm*) r_blas=PhiPack ;;
